package.json

```diff
@@ -29,9 +29,11 @@
     "browserify": "^12.0.1",
     "bundle-collapser": "^1.1.1",
     "coffee-script": "^1.8.0",
+    "core-js": "^2.2.1",
     "coveralls": "^2.11.6",
     "del": "^2.0.2",
     "derequire": "^2.0.3",
+    "es6-symbol": "^3.0.2",
```

Comment:
It's a leftover from a @puradox fix on tests — I will clean up this!

On Sat, Apr 9, 2016, 00:26 Dan Abramov <notifications@github.com> wrote:

> In package.json
> <https://github.com/facebook/react/pull/6377#discussion_r59097655>:
>
> >      "coveralls": "^2.11.6",
> >      "del": "^2.0.2",
> >      "derequire": "^2.0.3",
> > +    "es6-symbol": "^3.0.2",
>
> Is this necessary? Doesn’t appear to be used.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react/pull/6377/files/ad94295f139a8190286852e14a5c731065f5f0ba#r59097655>
>
-- 

Ryan Lahfa, de mon téléphone.